tools/modelgen/generators/templates/register.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ range .Objects }}
+func Register{{ initialCap .Name }}Callbacks(handler {{ initialCap .Name }}Callbacks) {
+  objCallbackHandler.{{ initialCap .Name }}Cb = handler
```

Comment:
yeah, this is returning an error now and may be it can invoke a default handler. 

I was suggesting to have a <>Cb initialized to a default no-op handler instead so that client is not forced to register callback for objects it doesn't care about.

This will also help when number of objects and clients grow/change.  We ideally don't want to change the clients that don't care whenever that happens.